posthog/tasks/test/test_session_recording_retention.py

```diff
@@ -46,11 +46,12 @@ def test_deletes_from_django(self) -> None:
             self.assertEqual(SessionRecordingEvent.objects.count(), 1)
             self.assertEqual(SessionRecordingEvent.objects.last(), event_after_threshold)
 
-    def create_snapshot(self, session_id: str, timestamp: datetime) -> SessionRecordingEvent:
+    def create_snapshot(self, session_id: str, timestamp: datetime, window_id: str = "3") -> SessionRecordingEvent:
```

Comment:
The default here imo makes less sense. Especially given it's optional (it won't exist for historic data), a better default would be None.